pkgs/by-name/mp/mpb/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  autoreconfHook,
+  gfortran,
+  blas,
+  lapack,
+  fftw,
+  hdf5,
+  libctl,
+  guile,
+}:
+
+assert !blas.isILP64;
+assert !lapack.isILP64;
+
+stdenv.mkDerivation rec {
+  pname = "mpb";
+  version = "1.11.1";
+
+  src = fetchFromGitHub {
+    owner = "NanoComp";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+2cMjZSGdfngtGoAeZRPRPBDvflTEIOWO8Se0W6jv9k=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    gfortran
+  ];
+
+  buildInputs = [
+    blas
+    lapack
+    fftw
+    hdf5
+    libctl
+    guile
+  ];
+
+  enableParallelBuilding = true;
+
+  configureFlags = [
+    "--with-libctl=no"
+    "--enable-shared"
+    "--enable-maintainer-mode"
+    "--disable-dependency-tracking"
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "MIT Photonic-Bands: computation of photonic band structures in periodic media";
+    homepage = "https://mpb.readthedocs.io/en/latest/";
+    license = licenses.gpl2Only;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [
+      sheevy
```

Comment:
This only has meaning if you add yourself to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix
That's okay if you wish to maintain that package long term. In that case make a separate commit, where you add yourself to that file. If not, it's also fine if you keep just me here. I will have a look on that package.